relay-server/src/actors/upstream.rs

```diff
@@ -728,6 +826,9 @@ pub trait UpstreamQuery: Serialize {
     fn priority() -> RequestPriority {
         RequestPriority::Low
     }
+    fn retry() -> bool {
```

Comment:
_nit_: Add an empty line. 

I am thinking that default behavior should be to retry all queries, and we only selectively disable where we have more appropriate logic. In fact, most queries have custom logic, but it's still "safer" to assume retries.

should retry:
- `SendOutcomes`
- `GetProjectIds`

custom logic:
- `GetProjectStates`
- `GetRelays`
- `RegisterRequest`
- `RegisterResponse`

Reply:
I was thinking that it would be safer to be explicit about when there are retries since this would be a change from the previous behaviour. If you go specifically to a message and mark it as retry then presumably you know what you are doing, as opposed to have everything change and then go and mark what shouldn't change.

Reply:
Ok, this makes sense. Let's keep it `false` by default, then.